.github/workflows/ci.yml

```diff
@@ -18,36 +18,25 @@ jobs:
       fail-fast: false
       matrix:
         suite:
-          - { path: bundler, name: bundler1, ci_node_total: 2, ci_node_index: 0, ecosystem: bundler }
-          - { path: bundler, name: bundler1, ci_node_total: 2, ci_node_index: 1, ecosystem: bundler }
-          - { path: bundler, name: bundler2, ci_node_total: 2, ci_node_index: 0, ecosystem: bundler }
-          - { path: bundler, name: bundler2, ci_node_total: 2, ci_node_index: 1, ecosystem: bundler }
+          - { path: bundler, name: bundler1, ecosystem: bundler }
+          - { path: bundler, name: bundler2, ecosystem: bundler }
           - { path: cargo, name: cargo, ecosystem: cargo }
           - { path: common, name: common, ecosystem: common}
-          - { path: composer, name: composer, ci_node_total: 2, ci_node_index: 0, ecosystem: composer }
-          - { path: composer, name: composer, ci_node_total: 2, ci_node_index: 1, ecosystem: composer }
+          - { path: composer, name: composer, ecosystem: composer }
           - { path: docker, name: docker, ecosystem: docker }
           - { path: elm, name: elm, ecosystem: elm }
           - { path: git_submodules, name: git_submodules, ecosystem: gitsubmodule }
           - { path: github_actions, name: github_actions, ecosystem: github-actions }
-          - { path: go_modules, name: go_module, ci_node_total: 2, ci_node_index: 0, ecosystem: gomod }
-          - { path: go_modules, name: go_module, ci_node_total: 2, ci_node_index: 1, ecosystem: gomod }
+          - { path: go_modules, name: go_module, ecosystem: gomod }
```

Comment:
How many different names can we have for the same thing?! Not the fault of this PR! 😄